packages/maskbook/src/plugins/dHEDGE/base.tsx

```diff
@@ -7,7 +7,7 @@ export const base: Plugin.Shared.Definition = {
     icon: <DHEDGEIcon />,
     name: { fallback: 'dHEDGE' },
     description: { fallback: 'Decentralized hedge funds on Ethereum.' },
-    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
+    publisher: { name: { fallback: 'iRhonin' }, link: 'https://github.com/iRhonin' },
```

Comment:
@Jack-Works i think need add a field, reference into bounty (issue)

Reply:
Why is that a necessary part of plugin information?